content/en/docs/reference/kubectl/overview.md

```diff
@@ -71,6 +71,20 @@ Flags that you specify from the command line override default values and any cor
 
 If you need help, run `kubectl help` from the terminal window.
 
+## Namespace Defaulting
```

Comment:
Important to call out that this only applies to in-cluster auth - so a running pod using kubectl.